content/chapter-04/NetworkManagementLab.md

```diff
@@ -1,11 +1,61 @@
 # Lab: Cloud Network Management
 
-## Objectives
-
 * Get familiar with Cloud Provider access control management
-* Review options available for VPN connectivity on the portal
+* You will create three small instances on this lab:
+  * **In-Access**: Accessible from outside your network on port 80
+  * **Middle-Access** Accessible only from **In-Access** and **Only-internal** on port 80
+  * **Only-internal** Accessible from **In-Access** and **Middle-Access**, but without access to the outside world
```

Comment:
**Internal Traffic** Open to **Inbound  PublicTraffic** and to **Private and Public Traffic**, no outbound traffic to the outside world

Reply:
Done